doc/crypto/EVP_PKEY_new.pod

```diff
@@ -37,8 +37,6 @@ used.
 EVP_PKEY_new() returns either the newly allocated B<EVP_PKEY> structure or
 B<NULL> if an error occurred.
 
-EVP_PKEY_up_ref() and EVP_PKEY_free() do not return a value.
```

Comment:
A description of what the return value from EVP_PKEY_up_ref() means is required here.